cmd/kube-proxy/app/options.go

```diff
@@ -416,17 +436,6 @@ func (o *Options) writeConfigFile() (err error) {
 	return nil
 }
 
-// addressFromDeprecatedFlags returns server address from flags
-// passed on the command line based on the following rules:
-// 1. If port is 0, disable the server (e.g. set address to empty).
```

Comment:
hm... do we still provide any way to do that? (other than setting the address to a non-matching cidr?)

Reply:
Currently, this is not possible for anyone using config to disable healthz server, config will get precedence over flags and default values will be used if unspecified in config.

We are creating healthzServer when `len(config.HealthzBindAddresses) > 0`, I can add a check `HealthzBindPort > 0`  but that can be breaking for someone who is running healthz address on a specific ip and random port.

Reply:
hm... I guess you could do something like `--metrics-bind-address 255.255.255.255`. As long as they didn't do `--bind-hard-fail`.